internal_worker.go

```diff
@@ -850,7 +851,8 @@ func newHostEnvironment() *hostEnvImpl {
 		workflowAliasMap: make(map[string]string),
 		activityFuncMap:  make(map[string]activity),
 		activityAliasMap: make(map[string]string),
-		encoding:         gobEncoding{},
+		encoding:         jsonEncoding{},
+		tEncoding:        thriftEncoding{},
```

Comment:
just find out that we never set the thriftEncoding (it is always nil), and since nobody complained about it, we are sure no one is using thrift encoding at this point.

Reply:
It could be using it directly.